### PR TITLE
fix: #714 cleanup frontend code

### PR DESCRIPTION
**Pull request by NickSaglioni**
1. Removed the unused components:
- Footer;
- Breadcrumb;
- Home;
- ManageAccess;
- SelectApplication.
2. Cleaned up all route paths
3. Test application is still working

### CODE REVIEW

**Comment by MCatherine1994** [comment]
Hi Nick, sorry I forgot to mention, maybe we want to check the tokens in the default-theme as well, I wonder if those are for the old bootstrap style? do we still need them? thanks!!

**Comment by NickSaglioni** [comment]
> Hi Nick, sorry I forgot to mention, maybe we want to check the tokens in the `frontend/src/assets/styles/default-theme.scss` as well, I wonder if those are for the old bootstrap style? do we still need them? Also this one `frontend/src/assets/styles/components/form-elements.scss` thanks!!
> 
> Also want to check the images, and packages in package.json

Assets and packages have been cleaned up.

File: frontend/package.json [hunk 1]
```diff
@@ -14,11 +14,6 @@
     },
     "dependencies": {
         "@bcgov-nr/nr-fsa-theme": "^1.1.0",
-        "@fortawesome/fontawesome-svg-core": "^6.2.1",
-        "@fortawesome/free-regular-svg-icons": "^6.2.1",
-        "@fortawesome/free-solid-svg-icons": "^6.2.1",
-        "@fortawesome/vue-fontawesome": "^3.0.2",
-        "@popperjs/core": "^2.11.6",
```
Comment (MCatherine1994) [inline frontend/package.json:21]: I'm not sure where we use this @popperjs/core package previously, just want to make sure we tested everything still works after removed it, thanks!!

**Comment by MCatherine1994** [comment]
let's merge from the main branch, and then do a clean install to get the clean package-lock.json

File: frontend/package.json (hunk 1 above)
Comment (NickSaglioni) [reply frontend/package.json:21]: PopperJS is for tooltip and pop-ups, we are not using anywhere in the new design.

**Review by MCatherine1994 — APPROVED** [review]

**Pull request merged**